packages/gluestack-ui/src/r5/feedback/fhir-error.tsx

```diff
@@ -0,0 +1,74 @@
+import { FhirErrorRendererProps } from "@bonfhir/react/r5";
+// importing from types because Alert type has identical name in themed lib
+import { IAlertProps } from "@gluestack-ui/alert/lib/typescript/types";
```

Comment:
I would suggest using `ComponentProps<typeof Alert>` instead of reaching out deep into the import path to pull out `IAlertProps` - as it might be more fragile to the library updates.

Reply:
Oh nice,  I didn't know about that. Good to know!